### ISSUE TITLE
Lightsaber Crash on Endor

### ISSUE THREAD

**Issue by darkknight2013** [Parzivail-Modding-Team/GalaxiesParzisStarWarsMod#32]
**Describe the bug**
Whilst on the forest moon of Endor, I activated my lightsaber. When I did so, the world and Minecraft crashed. Reopening this world was also impossible, as they would also crash when it was returned to.

**To Reproduce**
Steps to reproduce the behavior (example):
1. Travel to Endor (/cdim 3)
2. Wait for world to generate
3. Activate Lightsaber (crouch + right-click)
4. Crash

**Expected behavior**
If the following goes as expected, Minecraft should crash and shutdown immediately. Also, if this should take place, the map should also crash when you try to reopen it.

**Crash Report**
```
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 28/04/19 15:30
Description: Ticking entity

java.lang.IllegalArgumentException: value already present: com.parzivail.swg.audio.MovingSoundLightsaberIdle@704d01a8
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:238)
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:215)
	at net.minecraft.client.audio.SoundManager.func_148611_c(SoundManager.java:368)
	at net.minecraft.client.audio.SoundHandler.func_147682_a(SourceFile:154)
	at com.parzivail.util.audio.ClientSoundHandler.playSound(ClientSoundHandler.java:21)
	at com.parzivail.swg.proxy.Client.tickSounds(Client.java:285)
	at com.parzivail.swg.handler.EventHandler.on(EventHandler.java:460)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_EventHandler_on_PlayerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:345)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:220)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2006)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:238)
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:215)
	at net.minecraft.client.audio.SoundManager.func_148611_c(SoundManager.java:368)
	at net.minecraft.client.audio.SoundHandler.func_147682_a(SourceFile:154)
	at com.parzivail.util.audio.ClientSoundHandler.playSound(ClientSoundHandler.java:21)
	at com.parzivail.swg.proxy.Client.tickSounds(Client.java:285)
	at com.parzivail.swg.handler.EventHandler.on(EventHandler.java:460)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_EventHandler_on_PlayerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:345)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:220)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)

-- Entity being ticked --
Details:
	Entity Type: null (net.minecraft.client.entity.EntityClientPlayerMP)
	Entity ID: 262
	Entity Name: dark_knight_2013
	Entity's Exact location: -29.63, 100.54, -7.49
	Entity's Block location: World: (-30,100,-8), Chunk: (at 2,6,8 in -2,-1; contains blocks -32,0,-16 to -17,255,-1), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Entity's Momentum: 0.00, -0.08, 0.00
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1887)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['dark_knight_2013'/262, l='MpServer', x=-29.63, y=100.54, z=-7.49]]
	Chunk stats: MultiplayerChunkCache: 529, 529
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 24082 game time, 24082 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 2 total; [EntityZombie['Zombie'/194308, l='MpServer', x=-77.50, y=98.00, z=17.50], EntityClientPlayerMP['dark_knight_2013'/262, l='MpServer', x=-29.63, y=100.54, z=-7.49]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:919)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 456217800 bytes (435 MB) / 939524096 bytes (896 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1558 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1558} [Minecraft Forge] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	pswg{0.0.7} [Galaxies: Parzi's Star Wars Mod] (pswg-0.0.7-1.7.10.jar) 
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13521 Compatibility Profile Context 24.20.11020.2' Renderer: 'AMD Radeon(TM) R5 Graphics'
	Launched Version: 1.7.10-Forge10.13.4.1558-1.7.10
	LWJGL: 2.9.1
	OpenGL: AMD Radeon(TM) R5 Graphics GL version 4.5.13521 Compatibility Profile Context 24.20.11020.2, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)
```

**Please complete the following information:**
 - Mod Version 0.0.7
 - Minecraft Version 1.7.10
 - Forge Version 10.13.4.1558
 - Java Version 8

**Additional context**
There were no other mods installed.

**Comment by darkknight2013** [comment]
Just as an update, I've tested on another world and it crashed again. This time, I'd been to other dimensions, including Hyperspace and Hoth, and activated the Lightsaber on the default "Earth" dimension. Could it be linked with the /cdim command?


**Crash Report**
```
---- Minecraft Crash Report ----
// I just don't know what went wrong :(

Time: 28/04/19 17:45
Description: Ticking entity

java.lang.IllegalArgumentException: value already present: com.parzivail.swg.audio.MovingSoundLightsaberIdle@4c0f1c4b
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:238)
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:215)
	at net.minecraft.client.audio.SoundManager.func_148611_c(SoundManager.java:368)
	at net.minecraft.client.audio.SoundHandler.func_147682_a(SourceFile:154)
	at com.parzivail.util.audio.ClientSoundHandler.playSound(ClientSoundHandler.java:21)
	at com.parzivail.swg.proxy.Client.tickSounds(Client.java:285)
	at com.parzivail.swg.handler.EventHandler.on(EventHandler.java:460)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_EventHandler_on_PlayerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:345)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:220)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2006)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:238)
	at com.google.common.collect.HashBiMap.put(HashBiMap.java:215)
	at net.minecraft.client.audio.SoundManager.func_148611_c(SoundManager.java:368)
	at net.minecraft.client.audio.SoundHandler.func_147682_a(SourceFile:154)
	at com.parzivail.util.audio.ClientSoundHandler.playSound(ClientSoundHandler.java:21)
	at com.parzivail.swg.proxy.Client.tickSounds(Client.java:285)
	at com.parzivail.swg.handler.EventHandler.on(EventHandler.java:460)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_16_EventHandler_on_PlayerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:345)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:220)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)

-- Entity being ticked --
Details:
	Entity Type: null (net.minecraft.client.entity.EntityClientPlayerMP)
	Entity ID: 697
	Entity Name: dark_knight_2013
	Entity's Exact location: -136.79, 51.54, 203.26
	Entity's Block location: World: (-137,51,203), Chunk: (at 7,3,11 in -9,12; contains blocks -144,0,192 to -129,255,207), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Entity's Momentum: 0.00, -0.08, 0.00
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1887)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['dark_knight_2013'/697, l='MpServer', x=-136.79, y=51.54, z=203.26]]
	Chunk stats: MultiplayerChunkCache: 581, 581
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 7599 game time, 7599 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 134 total; [EntityZombie['Zombie'/825613, l='MpServer', x=-73.50, y=21.00, z=262.50], EntityZombie['Zombie'/825614, l='MpServer', x=-74.50, y=21.00, z=259.50], EntityCreeper['Creeper'/826895, l='MpServer', x=-112.39, y=38.00, z=215.95], EntitySkeleton['Skeleton'/826887, l='MpServer', x=-102.50, y=24.00, z=254.50], EntityEnderman['Enderman'/2319, l='MpServer', x=-129.95, y=48.00, z=219.83], EntityZombie['Zombie'/2326, l='MpServer', x=-120.78, y=28.00, z=233.59], EntityZombie['Zombie'/827412, l='MpServer', x=-125.00, y=31.00, z=138.41], EntityBat['Bat'/826391, l='MpServer', x=-103.17, y=29.08, z=248.57], EntityBat['Bat'/827152, l='MpServer', x=-213.35, y=39.53, z=257.40], EntityZombie['Zombie'/2333, l='MpServer', x=-97.84, y=19.00, z=256.28], EntityZombie['Zombie'/2334, l='MpServer', x=-102.50, y=24.00, z=251.50], EntitySheep['Sheep'/36, l='MpServer', x=-201.50, y=79.00, z=156.50], EntitySheep['Sheep'/37, l='MpServer', x=-202.50, y=80.00, z=153.50], EntityZombie['Zombie'/827177, l='MpServer', x=-182.50, y=29.00, z=192.50], EntitySheep['Sheep'/38, l='MpServer', x=-203.50, y=77.00, z=156.50], EntityZombie['Zombie'/827178, l='MpServer', x=-177.50, y=29.00, z=193.50], EntitySheep['Sheep'/39, l='MpServer', x=-201.50, y=79.00, z=156.50], EntitySheep['Sheep'/40, l='MpServer', x=-200.50, y=72.00, z=166.78], EntityZombie['Zombie'/827172, l='MpServer', x=-89.50, y=16.00, z=218.50], EntitySheep['Sheep'/41, l='MpServer', x=-202.50, y=67.00, z=166.50], EntitySheep['Sheep'/42, l='MpServer', x=-200.50, y=73.00, z=165.47], EntitySheep['Sheep'/43, l='MpServer', x=-200.50, y=70.00, z=167.50], EntitySkeleton['Skeleton'/827425, l='MpServer', x=-89.50, y=28.00, z=210.50], EntitySkeleton['Skeleton'/827426, l='MpServer', x=-87.53, y=29.00, z=215.69], EntityEnderman['Enderman'/827427, l='MpServer', x=-91.69, y=28.00, z=209.22], EntityBat['Bat'/827454, l='MpServer', x=-83.60, y=23.12, z=243.34], EntityBat['Bat'/827455, l='MpServer', x=-85.30, y=23.03, z=245.60], EntitySheep['Sheep'/52, l='MpServer', x=-184.50, y=86.00, z=141.50], EntitySkeleton['Skeleton'/826936, l='MpServer', x=-120.16, y=44.00, z=215.03], EntityZombie['Zombie'/827192, l='MpServer', x=-111.91, y=19.00, z=189.50], EntitySheep['Sheep'/53, l='MpServer', x=-184.50, y=86.00, z=141.50], EntityCreeper['Creeper'/827193, l='MpServer', x=-111.50, y=18.00, z=190.50], EntitySkeleton['Skeleton'/827449, l='MpServer', x=-129.06, y=45.00, z=161.56], EntityWitch['Witch'/1590, l='MpServer', x=-119.63, y=22.00, z=212.63], EntitySheep['Sheep'/54, l='MpServer', x=-184.50, y=85.00, z=144.50], EntitySkeleton['Skeleton'/827450, l='MpServer', x=-82.50, y=26.00, z=250.50], EntitySheep['Sheep'/55, l='MpServer', x=-186.09, y=86.00, z=144.91], EntityBat['Bat'/825915, l='MpServer', x=-92.35, y=27.03, z=204.31], EntitySpider['Spider'/827190, l='MpServer', x=-107.00, y=18.00, z=194.75], EntityZombie['Zombie'/827191, l='MpServer', x=-112.50, y=18.00, z=191.50], EntityCow['Cow'/64, l='MpServer', x=-176.50, y=64.00, z=272.53], EntityCow['Cow'/65, l='MpServer', x=-175.50, y=68.00, z=272.50], EntityCow['Cow'/66, l='MpServer', x=-177.53, y=64.00, z=270.47], EntityCow['Cow'/67, l='MpServer', x=-176.19, y=63.00, z=271.38], EntityItem['item.tile.mushroom'/68, l='MpServer', x=-161.25, y=17.13, z=194.97], EntityCow['Cow'/69, l='MpServer', x=-141.41, y=67.00, z=237.31], EntityCow['Cow'/70, l='MpServer', x=-129.75, y=68.00, z=225.69], EntityBat['Bat'/827466, l='MpServer', x=-172.22, y=17.85, z=186.56], EntityCow['Cow'/71, l='MpServer', x=-134.69, y=63.00, z=236.78], EntityBat['Bat'/827467, l='MpServer', x=-166.36, y=18.37, z=182.54], EntityCow['Cow'/72, l='MpServer', x=-136.81, y=64.00, z=229.09], EntityCow['Cow'/73, l='MpServer', x=-137.22, y=63.00, z=263.78], EntityCow['Cow'/74, l='MpServer', x=-137.91, y=63.00, z=255.13], EntityBat['Bat'/825670, l='MpServer', x=-77.25, y=25.10, z=185.31], EntityCow['Cow'/75, l='MpServer', x=-139.53, y=63.00, z=263.53], EntityCow['Cow'/76, l='MpServer', x=-139.53, y=63.00, z=263.53], EntityPig['Pig'/85, l='MpServer', x=-120.78, y=63.00, z=206.91], EntityPig['Pig'/86, l='MpServer', x=-107.28, y=67.00, z=197.16], EntitySpider['Spider'/826714, l='MpServer', x=-113.50, y=31.00, z=134.50], EntityPig['Pig'/87, l='MpServer', x=-145.91, y=66.00, z=223.78], EntityPig['Pig'/88, l='MpServer', x=-126.50, y=64.00, z=196.25], EntityWitch['Witch'/826711, l='MpServer', x=-150.50, y=17.00, z=214.50], EntityPig['Pig'/97, l='MpServer', x=-110.50, y=63.00, z=243.50], EntityPig['Pig'/98, l='MpServer', x=-108.50, y=64.00, z=243.50], EntityCreeper['Creeper'/826990, l='MpServer', x=-90.50, y=39.00, z=250.50], EntityPig['Pig'/99, l='MpServer', x=-109.50, y=63.00, z=240.50], EntityPig['Pig'/100, l='MpServer', x=-110.50, y=63.00, z=243.50], EntityCow['Cow'/101, l='MpServer', x=-95.44, y=88.00, z=127.72], EntityCow['Cow'/102, l='MpServer', x=-100.53, y=89.00, z=126.19], EntityCow['Cow'/105, l='MpServer', x=-96.50, y=86.00, z=144.78], EntityCow['Cow'/106, l='MpServer', x=-96.50, y=86.00, z=143.19], EntityCow['Cow'/107, l='MpServer', x=-95.50, y=84.00, z=145.50], EntityCow['Cow'/108, l='MpServer', x=-92.31, y=74.00, z=149.31], EntityZombie['Zombie'/826720, l='MpServer', x=-104.09, y=24.00, z=256.75], EntityBat['Bat'/827234, l='MpServer', x=-99.75, y=15.53, z=256.79], EntityZombie['Zombie'/825699, l='MpServer', x=-135.50, y=27.00, z=198.97], EntityBat['Bat'/827235, l='MpServer', x=-138.25, y=50.10, z=227.75], EntityCow['Cow'/113, l='MpServer', x=-81.50, y=69.00, z=130.50], EntityCow['Cow'/114, l='MpServer', x=-79.50, y=75.00, z=128.50], EntityBat['Bat'/827262, l='MpServer', x=-184.53, y=14.10, z=185.25], EntityCow['Cow'/115, l='MpServer', x=-80.22, y=77.00, z=127.47], EntityCow['Cow'/116, l='MpServer', x=-81.53, y=77.00, z=128.81], EntitySkeleton['Skeleton'/827508, l='MpServer', x=-137.50, y=18.00, z=177.50], EntitySkeleton['Skeleton'/827510, l='MpServer', x=-132.50, y=20.00, z=258.50], EntitySkeleton['Skeleton'/827511, l='MpServer', x=-133.50, y=20.00, z=259.50], EntitySquid['Squid'/892, l='MpServer', x=-78.97, y=60.22, z=241.06], EntitySquid['Squid'/894, l='MpServer', x=-76.94, y=59.88, z=242.34], EntityClientPlayerMP['dark_knight_2013'/697, l='MpServer', x=-136.79, y=51.54, z=203.26], EntityBat['Bat'/827269, l='MpServer', x=-105.66, y=22.10, z=134.31], EntityZombie['Zombie'/825219, l='MpServer', x=-203.50, y=38.00, z=206.50], EntitySquid['Squid'/826780, l='MpServer', x=-76.84, y=59.97, z=179.41], EntitySquid['Squid'/826782, l='MpServer', x=-79.50, y=58.66, z=177.16], EntitySquid['Squid'/826776, l='MpServer', x=-81.50, y=61.16, z=186.47], EntitySquid['Squid'/826778, l='MpServer', x=-79.69, y=59.88, z=173.75], EntityBat['Bat'/827290, l='MpServer', x=-75.39, y=18.83, z=250.19], EntitySpider['Spider'/826260, l='MpServer', x=-106.03, y=31.00, z=130.19], EntitySquid['Squid'/826775, l='MpServer', x=-80.25, y=60.28, z=190.41], EntityCreeper['Creeper'/826257, l='MpServer', x=-101.66, y=31.00, z=123.50], EntityCreeper['Creeper'/825747, l='MpServer', x=-83.50, y=32.00, z=226.50], EntityItem['item.item.rottenFlesh'/11426, l='MpServer', x=-80.13, y=36.13, z=210.56], EntityCreeper['Creeper'/825518, l='MpServer', x=-88.50, y=28.00, z=257.50], EntityCreeper['Creeper'/825768, l='MpServer', x=-130.44, y=23.00, z=217.44], EntitySkeleton['Skeleton'/827051, l='MpServer', x=-113.50, y=23.00, z=214.50], EntityCreeper['Creeper'/827045, l='MpServer', x=-70.50, y=33.00, z=269.50], EntityCreeper['Creeper'/825766, l='MpServer', x=-129.03, y=25.00, z=231.66], EntityCreeper['Creeper'/827046, l='MpServer', x=-71.50, y=33.00, z=268.50], EntityCreeper['Creeper'/825767, l='MpServer', x=-142.16, y=21.00, z=208.75], EntitySquid['Squid'/826784, l='MpServer', x=-80.22, y=61.38, z=169.28], EntityCreeper['Creeper'/827043, l='MpServer', x=-146.50, y=28.00, z=239.50], EntitySkeleton['Skeleton'/11699, l='MpServer', x=-117.09, y=26.00, z=211.50], EntityEnderman['Enderman'/827058, l='MpServer', x=-121.50, y=17.00, z=239.50], EntitySkeleton['Skeleton'/2497, l='MpServer', x=-124.03, y=31.00, z=138.50], EntityBat['Bat'/827341, l='MpServer', x=-134.53, y=30.10, z=232.75], EntityZombie['Zombie'/827100, l='MpServer', x=-110.00, y=35.00, z=164.56], EntityZombie['Zombie'/827101, l='MpServer', x=-98.50, y=38.00, z=165.50], EntityItem['item.item.reeds'/826328, l='MpServer', x=-152.34, y=63.13, z=163.09], EntityItem['item.item.reeds'/826329, l='MpServer', x=-150.78, y=63.13, z=163.16], EntityZombie['Zombie'/827098, l='MpServer', x=-100.50, y=38.00, z=164.50], EntityBat['Bat'/1239, l='MpServer', x=-145.38, y=46.03, z=198.59], EntityBat['Bat'/1241, l='MpServer', x=-128.53, y=17.10, z=265.28], EntitySpider['Spider'/825814, l='MpServer', x=-119.72, y=25.00, z=217.53], EntitySkeleton['Skeleton'/825811, l='MpServer', x=-121.84, y=25.00, z=221.22], EntityZombie['Zombie'/1251, l='MpServer', x=-79.50, y=28.00, z=258.50], EntitySkeleton['Skeleton'/827112, l='MpServer', x=-206.50, y=12.00, z=233.50], EntityCreeper['Creeper'/826601, l='MpServer', x=-101.50, y=31.00, z=126.69], EntityCreeper['Creeper'/826598, l='MpServer', x=-101.44, y=31.00, z=125.75], EntityCreeper['Creeper'/826599, l='MpServer', x=-106.41, y=31.00, z=129.00], EntityCreeper['Creeper'/827385, l='MpServer', x=-77.90, y=16.00, z=244.56], EntityCreeper['Creeper'/827125, l='MpServer', x=-156.50, y=16.00, z=202.50], EntityZombie['Zombie'/827381, l='MpServer', x=-85.50, y=13.00, z=244.50], EntitySkeleton['Skeleton'/827126, l='MpServer', x=-166.50, y=16.00, z=194.50], EntityZombie['Zombie'/827382, l='MpServer', x=-86.53, y=14.00, z=249.09], EntityZombie['Zombie'/827383, l='MpServer', x=-84.50, y=13.00, z=247.50], EntityCreeper['Creeper'/827122, l='MpServer', x=-98.50, y=33.00, z=187.50]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:919)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 634188616 bytes (604 MB) / 1040187392 bytes (992 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1558 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1558} [Minecraft Forge] (forge-1.7.10-10.13.4.1558-1.7.10.jar) 
	UCHIJAAAA	pswg{0.0.7} [Galaxies: Parzi's Star Wars Mod] (pswg-0.0.7-1.7.10.jar) 
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13521 Compatibility Profile Context 24.20.11020.2' Renderer: 'AMD Radeon(TM) R5 Graphics'
	Launched Version: 1.7.10-Forge10.13.4.1558-1.7.10
	LWJGL: 2.9.1
	OpenGL: AMD Radeon(TM) R5 Graphics GL version 4.5.13521 Compatibility Profile Context 24.20.11020.2, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)
```

**Comment by parzivail** [comment]
No, it has to do with the lightsaber ignition. If it starts trying to play a sound that it thinks is already playing, it crashes. The problem is that the two ideas can get out of sync and it will get into a bad state where trying to play a sound that isn't actually playing fails because the sound manager still believes its playing. This should theoretically be fixed in my sound handler for 1.12+ releases, but I haven't verified that.

**Comment by parzivail** [comment]
This is defnitely fixed now in the alphas so I'm going to close this now.
